src/components/MenuButton/MenuButton.tsx

```diff
@@ -153,7 +166,6 @@ export default class MenuButton extends React.Component<MenuButtonProps> {
         border: 0,
         padding: 0,
         opacity: 0.7,
-        width: `${this.props.iconSize}px`,
```

Comment:
We tested this and the hardcoded width didn't seem to make a difference. Fabric will by default make the width the size of the inner content.